src/ensembl/package.json

```diff
@@ -118,13 +141,10 @@
     "jest": "26.6.3",
     "jest-fetch-mock": "3.0.3",
     "jest-transform-stub": "2.0.0",
-    "koa": "2.13.1",
-    "koa-connect": "2.1.0",
-    "koa-proxy": "1.0.0-alpha.3",
-    "koa-static": "5.0.0",
```

Comment:
Since our prod test server was using `koa`, we'll need to rewrite `server.js` to get it to work again

Reply:
We won't need that server.js anymore. If I forgot to delete it, I will.

All the server-side code is now in the `src/server` folder

Reply:
Yes it is still there along with the `setup-ssl` script. We will need to modify `serve:prod` in `npm` too right?

Reply:
Removed the `server.js` and `setup-ssl.js` files, along with the npm scripts calling them in the `package.json`